src/pages/documentation.html

```diff
@@ -139,8 +139,11 @@
 					</div>
 				</div>
 			</div>
-			<div class="section mod_comparison">
+			<div class="section mod_comparison color_lightblue">
 				<div class="section__content">
+					<div class="tile__icon">
```

Comment:
inner part of a component can't be used without the component itself

Reply:
1. Why do we need this icon in this block?

2. The background under the icon is conflicting with the background under the section. Additionally, I kindly ask to fix the following visual bugs:
![123](https://user-images.githubusercontent.com/748043/52414418-8b54a080-2ae4-11e9-9fac-8a3e0f354cf6.png)

Reply:
1. Because in Projects there is too an icon. After speaking with Micheal we decided to add it, so it looks similar.

2. The position of the icon I am currently fixing but I am still waiting for help. Should be done tomorrow